metadata/io.undertow/undertow-core/2.3.0.Final/jni-config.json

```diff
@@ -0,0 +1,45 @@
+[
+{
+  "condition":{"typeReachable":"io.undertow.Undertow"},
+  "name":"[Lcom.sun.management.internal.DiagnosticCommandArgumentInfo;"
+},
+{
+  "condition":{"typeReachable":"io.undertow.Undertow"},
+  "name":"[Lcom.sun.management.internal.DiagnosticCommandInfo;"
+},
+{
+  "condition":{"typeReachable":"io.undertow.Undertow"},
+  "name":"com.sun.management.internal.DiagnosticCommandArgumentInfo",
+  "methods":[{"name":"<init>","parameterTypes":["java.lang.String","java.lang.String","java.lang.String","java.lang.String","boolean","boolean","boolean","int"] }]
+},
+{
+  "condition":{"typeReachable":"io.undertow.Undertow"},
+  "name":"com.sun.management.internal.DiagnosticCommandInfo",
+  "methods":[{"name":"<init>","parameterTypes":["java.lang.String","java.lang.String","java.lang.String","java.lang.String","java.lang.String","java.lang.String","boolean","java.util.List"] }]
+},
+{
+  "condition":{"typeReachable":"io.undertow.Undertow"},
+  "name":"java.lang.String",
+  "methods":[{"name":"lastIndexOf","parameterTypes":["int"] }, {"name":"substring","parameterTypes":["int"] }]
+},
+{
+  "condition":{"typeReachable":"io.undertow.Undertow"},
+  "name":"java.lang.System",
+  "methods":[{"name":"getProperty","parameterTypes":["java.lang.String"] }, {"name":"setProperty","parameterTypes":["java.lang.String","java.lang.String"] }]
```

Comment:
Not sure if this is really needed. I think that whole jni-config can be removed.